allauth/socialaccount/forms.py

```diff
@@ -63,3 +63,6 @@ def save(self):
         signals.social_account_removed.send(
             sender=SocialAccount, request=self.request, socialaccount=account
         )
+        get_adapter().send_notification_mail(
+            "socialaccount/email/social_account_disconnected", self.request.user, {}
```

Comment:
Consistency: the already existing messages are named `socialaccount/messages/account_connected.txt` so we should drop the `social_` prefix here for sake of consistency.